semantic-conventions/src/opentelemetry/semconv/templating/markdown/__init__.py

```diff
@@ -216,16 +214,15 @@ def to_markdown_metric_table(
         """
         if not isinstance(semconv, MetricSemanticConvention):
             raise ValueError(
-                "semconv `{}` was specified with `metric_table`, but it is not a metric convention".format(
-                    semconv.semconv_id
-                )
+                f"semconv `{semconv.semconv_id}` was specified with `metric_table`, but it is not a metric convention"
             )
 
         output.write(
             "| Name     | Instrument Type | Unit (UCUM) | Description    |\n"
             "| -------- | --------------- | ----------- | -------------- |\n"
         )
         output.write(
+            # pylint: disable=C0209
```

Comment:
Instead of disabling, maybe store the complex expression in a named variable to make the code a bit more clear. If disabling, please don't disable by code, but with the readable identifier.

Reply:
Good idea, Done.